tests/test_basic.py

```
@@ -0,0 +1,124 @@
+            updated = self.app.task_manager.get_task(tid)
+            self.assertEqual(updated.status, 'COMPLETED')
+
+    def test_delete_task(self):
+        """Prüft, ob das Löschen einer Aufgabe funktioniert."""
+        with self.app.app_context():
+            task = self.app.task_manager.add_task('Loeschen')
+            tid = task.id
+        resp = self.client.post(f'/task/{tid}/delete')
+        self.assertEqual(resp.status_code, 302)
+        with self.app.app_context():
+            t1 = current_app.task_manager.add_task('A')
```

Ruff (0.12.2)

Undefined name `current_app`

(F821)